testproject/testproject/settings.py

```diff
@@ -4,16 +4,19 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',  # Add 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': 'testproject_db',                # Or path to database file if using sqlite3.
-        'USER': '',                              # Not used with sqlite3.
-        'PASSWORD': '',                          # Not used with sqlite3.
-        'HOST': '',                              # Set to empty string for localhost. Not used with sqlite3.
-        'PORT': '',                              # Set to empty string for default. Not used with sqlite3.
+        'ENGINE': 'django.db.backends.sqlite3',
+        # Add 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'testproject_db',  # Or path to database file if using sqlite3.
+        'USER': '',  # Not used with sqlite3.
+        'PASSWORD': '',  # Not used with sqlite3.
+        'HOST': '',  # Set to empty string for localhost. Not used with sqlite3.
+        'PORT': '',  # Set to empty string for default. Not used with sqlite3.
     }
 }
 
 # Use a fast hasher to speed up tests.
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.MD5PasswordHasher',
 ]
+
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
Instead of modifying the test project I believe fiber will need an `AppConfig` that defines the `default_auto_field` for fiber. Otherwise projects using a different value for `DEFAULT_AUTO_FIELD` will end up with a migration for django-fiber. See: https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default_auto_field